src/test/scala/ndrcApplication/utils/Configuration.scala

```diff
@@ -42,45 +39,41 @@ object Configuration {
     }
   }
 
+  private val ndrcBaseUrl: String =
+    "/apply-for-repayment-of-import-duty-and-import-vat/what-do-you-want-to-do"
+  private val authLogin: String =
+    "/auth-login-stub/gg-sign-in"
+
   lazy val settings: Configuration = create()
 
   def create(): Configuration = {
     environment match {
       case Environment.local =>
         new Configuration(
-          authLogin = "http://localhost:9949/auth-login-stub/gg-sign-in",
-          ndrcBaseUrl = s"$localhost$ndrcBaseUrl",
-          ndrcApplicationLandingUrl = s"$localhost$ndrcBaseUrl/what-do-you-want-to-do",
+          authLoginUrl = s"http://localhost:9949$authLogin",
+          ndrcApplicationLandingUrl = s"http://localhost:8450$ndrcBaseUrl",
           timeout = 10
         )
       case Environment.qa =>
         new Configuration(
-          authLogin = s"$qaUrl/auth-login-stub/gg-sign-in",
-          ndrcBaseUrl = s"$qaUrl$ndrcBaseUrl",
-          ndrcApplicationLandingUrl = s"$ndrcBaseUrl/what-do-you-want-to-do",
+          authLoginUrl = s"https://www.qa.tax.service.gov.uk$authLogin",
+          ndrcApplicationLandingUrl = ndrcBaseUrl,
           timeout = 10
         )
```

Comment:
Probably QA can be completely removed, because we won't be using tests on that environment, but I know it's not a part of this ticket so approved anyway.

Reply:
I agree with Patryk but approved anyway 👍

Reply:
I did think that, will get rid of it